pkg/utils/utils.go

```diff
@@ -115,6 +118,11 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState) error {
 		for _, iface := range newState.Spec.Interfaces {
 			if iface.PciAddress == ifaceStatus.PciAddress {
 				configured = true
+				if iface.EswitchMode == sriovnetworkv1.ESWITCHMODE_SWITCHDEV {
```

Comment:
@pliurh  this part will be updated in the following PR, right? e.g. To not skip switchdev device in daemon.

Reply:
removed.